package.json

```diff
@@ -71,7 +71,7 @@
       "git add"
     ],
     "src/**/*.go": [
-      "golangci-lint run -c .golangci.yml --fix",
+      "./aio/scripts/pre-commit-golangci-lint.sh",
```

Comment:
If I change 10 files, will it run 10 times the same script?

Reply:
lint-staged passes all staged filepaths at once.
So this script will be called only once.

Reply:
Ah, sorry. Didn't notice.